includes/class-webmention-mf2-handler.php

```diff
@@ -0,0 +1,389 @@
+<?php
+/**
+ * takes MF2 JSON and processes it
+ *
+ */
+class Webmention_MF2_Handler {
```

Comment:
Does it make sense to have an object here instead of a simple class? I only use classes for stateless libs.

Reply:
This is based a similar class in Semantic Linkbacks...but  so far it is just helping finding of representative author and item, stripping everything else. It could be an object...but this is more a stateless libs, as you described.